frameit/lib/frameit/editor.rb

```diff
@@ -238,12 +245,13 @@ def put_title_into_background(background)
     end
 
     def actual_font_size
-      [@image.width / 10.0].max.round
+      font_scale_factor = fetch_config['font_scale_factor'] || 0.1
+      [@image.width * font_scale_factor].max.round
     end
 
     # The space between the keyword and the title
     def keyword_padding
-      (actual_font_size / 2.0).round
+      (actual_font_size / 3.0).round
```

Comment:
Honestly, I see the too-small spacing as a bug (#7346). Since this PR fixes all kinds of other layout bugs, there will be a layout change (to the better) anyway. It's your code so you decide of course, but I'd advise against adding just another layout config parameter. It's hard to explain and I expect that most users are just fine with the current setting. I think the scope of frameit is not to invent another language like PDF, isn't it?